packages/slice-machine/lib/consts.ts

```diff
@@ -2,7 +2,6 @@
 export const RESERVED_SLICE_NAME = ["components"];
 
 export const DEFAULT_VARIATION_ID = "default";
-export const changelogPath = "changelog/versions";
```

Comment:
is this no longer used?

Reply:
No, there were no references for `changelogPath`. It may have been used in the old `server` directory, but it was replaced by the `start-slicemachine` package.